src/tabnote.ts

```diff
@@ -213,38 +213,41 @@ export class TabNote extends StemmableNote {
 
   // Get the default stem extension for the note
   getStemExtension(): number {
-    const glyphProps = this.getGlyphProps();
-
     if (this.stemExtensionOverride != null) {
       return this.stemExtensionOverride;
     }
 
-    if (glyphProps) {
-      return this.getStemDirection() === Stem.UP
-        ? glyphProps.tabnoteStemUpExtension
-        : glyphProps.tabnoteStemDownExtension;
+    return this.flag.getHeight() > Stem.HEIGHT ? this.flag.getHeight() - Stem.HEIGHT : 0;
+  }
+
+  static tabToElement(fret: string, scale: number = 1.0): Element {
+    let el: Element;
+
+    if (fret.toUpperCase() === 'X') {
+      el = new Element('TabNote');
+      el.setText(String.fromCodePoint(0xe263 /*accidentalDoubleSharp*/));
+      el.measureText();
+    } else {
+      el = new Element('TabNote.text');
```

Comment:
Here is the font size.

Reply:
? I don't see where the font size is specified here.

Reply:
In tables:
```
  TabNote: {
    text: {
      fontSize: 9,
    },
  },
```